R/merge.R

```diff
@@ -17,9 +17,9 @@ merge.data.table = function(x, y, by = NULL, by.x = NULL, by.y = NULL, all = FAL
     if (x0 && y0)
       warningf("Neither of the input data.tables to join have columns.")
     else if (x0)
-      warningf("Input data.table '%s' has no columns.", "x")
+      warningf("Input data.table x has no columns.")
```

Comment:
Why this change? This code is structured intentionally so that there are not two nearly-but-not-quite identical messages for translators.